xmr-btc/tests/e2e.rs

```diff
@@ -18,6 +18,8 @@ mod harness;
 const TEN_XMR: u64 = 10_000_000_000_000;
 const RELATIVE_REFUND_TIMELOCK: u32 = 1;
 const RELATIVE_PUNISH_TIMELOCK: u32 = 1;
+const ALICE_TEST_DB_FOLDER: &str = "../target/e2e-test-alice-recover";
```

Comment:
If you don't have a real reason for putting these db files into target then I would go for a tmp folder instead. I've introduced this dependency already in my PR so you can just use this:
e.g. 

```
use tempfile::tempdir;
// Create a directory inside of `std::env::temp_dir()`.
let alice_db_dir = tempdir()?;
let bob_db_dir = tempdir()?;
```


Same applies to a similar usage further down.